<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">background-color: Wheat</string>
     </property>
     <widget class="QPushButton" name="Screen1pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nie klikaj mnie!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">background-color: SkyBlue</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>i co zrobiłeś?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="screen2pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>300</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Powrót</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="mainPushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ekran główny</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
